.github/workflows/integration-tests.yml

```diff
@@ -284,3 +284,4 @@ jobs:
         run: ./integration_tests/integration_tests.test -test.failfast -test.v -test.run IntegrationTestSuite -testify.m Test${{ matrix.test_type }}
         env:
           ARCHIVE_NODE_URL: https://eth-mainnet.alchemyapi.io/v2/${{ secrets.ALCHEMY_ID }}
+          E2E_SKIP_CLEANUP: true
```

Comment:
Some permission issue when removing the folder when cleaning up. Bypassing that way for now

Reply:
could not cleaning cause any issues?